### PR TITLE
Makes a thing use QDEL_NULL

### PR DESCRIPTION
**Pull request by Cyberboss**
@ChangelingRain

### CODE REVIEW

File: code/game/machinery/doors/airlock.dm [hunk 1]
```diff
@@ -199,8 +199,7 @@
 	qdel(src)
 
 /obj/machinery/door/airlock/Destroy()
-	qdel(wires)
-	wires = null
+	QDEL_NULL(wires)
 	if(charge)
 		qdel(charge)
```
Comment (ChangelingRain) [inline code/game/machinery/doors/airlock.dm]: this too

Comment (Cyberboss) [inline code/game/machinery/doors/airlock.dm]: That's in rhe other PR

Comment (ChangelingRain) [inline code/game/machinery/doors/airlock.dm]: you need to do it in another pr you've set precedent

Comment (Cyberboss) [inline code/game/machinery/doors/airlock.dm]: But that PR refactored charge so it's valid :B
